### DISCUSSION TITLE
Which QoS standard does the AI-on-the-edge device use to communicate via MQTT?

### DISCUSSION THREAD

**Discussion by DasEiserneO** [jomjol/AI-on-the-edge-device#2694] [Q&A] [answered]
Which QoS standard does the AI-on-the-edge device use to communicate via MQTT?

**Comment by caco3** [comment, marked as answer]
This is hard coded to `QoS=1`, see https://github.com/jomjol/AI-on-the-edge-device/blob/rolling/code/components/jomjol_flowcontroll/ClassFlowMQTT.cpp#L238

**Reply by DasEiserneO** [reply]
Could it be possible to communicate with QoS 2?

**Reply by caco3** [reply]
Why do you need QoS=2?

Looking on https://www.hivemq.com/blog/mqtt-essentials-part-6-mqtt-quality-of-service-levels/#5 I do not see the point for QoS=2 on this system!
